internal/dockercompose/client.go

```diff
@@ -48,12 +48,7 @@ func NewDockerComposeClient(env docker.Env) DockerComposeClient {
 }
 
 func (c *cmdDCClient) Up(ctx context.Context, configPath string, serviceName model.TargetName, shouldBuild bool, stdout, stderr io.Writer) error {
-	var args []string
```

Comment:
why did this change in this pr? this doesn't seem right?

Reply:
oh, this was just me making the docker compose logs shut up because they make integration test output super gross. if you object i can put this back.

Reply:
i'm not sure...though as a user, i would probably expect --verbose to also turn on docker-compose --verbose?

Reply:
hmm. that's reasonable. i just wish `docker-compose --verbose` wasn't SO DAMN VERBOSE...

Reply:
i put this back